unicoremc/models.py

```diff
@@ -270,6 +259,24 @@ def to_dict(self):
             'docker_cmd': self.docker_cmd or '',
         }
 
+    def get_website_manager(self):
+        if not (self.application_type and self.application_type.project_type):
+            raise exceptions.ProjectTypeRequiredException(
+                'project_type is required')
+
+        # For non-standalone apps, we use the IogtWebsiteManager
+        if not self.own_repo():
+            return IogtWebsiteManager(self)
```

Comment:
:) I'm glad. I actually think I should rename this manger to be called `AggregatorWebsiteManager` to avoid the ambiguity. IoGT is technically not a correct term to refer to the non-standalone apps.